js/stringProcessing/getSentences.js

```diff
@@ -10,15 +10,18 @@ var getSubheadings = require( "./getSubheadings.js" ).getSubheadings;
 // All characters that indicate a sentence delimiter.
 var sentenceDelimiters = ".?!:;";
 
+// If a sentence delimiter is followed by one of these characters, it is a valid ending.
+var sentenceEndingRegex = /[\s<\)\]\"\']/;
+
 /**
- * Checks if the period is followed with a whitespace. If not, it is no ending of a sentence.
+ * Checks if the period is followed with a character that is a valid sentence ending. If not, it is no ending of a sentence.
  *
  * @param {string} text The text to split in sentences.
  * @param {number} index The current index to look for.
  * @returns {boolean} True if it doesn't match a whitespace.
  */
-var invalidateOnWhiteSpace = function( text, index ) {
-	return text.substring( index, index + 1 ).match( /\s/ ) === null;
+var invalidateOnCharacter = function(text, index ) {
+	return text.substring( index, index + 1 ).match( sentenceEndingRegex ) === null;
```

Comment:
Perhaps moving this into a generic method that takes a string and matches it like so  `return text.match( sentenceEndingRegex ) === nulll;` would be nicer. This would also allow you to reuse it on line 42.